src/main/java/hellfirepvp/astralsorcery/common/util/nbt/NBTHelper.java

```diff
@@ -354,8 +354,9 @@ public static FluidStack getFluid(CompoundNBT compound, String tag) {
     }
 
     public static void removeUUID(CompoundNBT compound, String key) {
-        compound.remove(key + "Most");
```

Comment:
No tag exists with "Most" added to the end of it when considering tools, weapons and armour.